addons/survey/static/src/scss/survey_templates_form.scss

```
@@ -19,14 +19,18 @@ div.o_frontend_to_backend_nav {
     height: 100%;
     overflow: auto;
     transition: box-shadow 0.3s ease-in-out;
-    box-shadow: inset 0 0 0 10000px rgba(255,255,255,.7);
+    box-shadow: inset 0 0 0 10000px rgba(255,255,255,0);
     background: no-repeat fixed center;
     background-size: cover;
     &.o_survey_background_transition {
         box-shadow: inset 0 0 0 10000px rgba(255,255,255,1);
     }
 }
 
+.o_survey_background_shadow {
+    box-shadow: inset 0 0 0 10000px rgba(255,255,255,.7) !important;
+}
```

I think this could be written:

```suggestion
    &.o_survey_background_shadow {
        box-shadow: inset 0 0 0 10000px rgba(255,255,255,.7); 
    }
    &.o_survey_background_transition {
        box-shadow: inset 0 0 0 10000px rgba(255,255,255,1);
    }
}
```
(with `.o_survey_background_transition` after so it is not overriden by `.o_survey_background_shadow` style as is the case with current version)

---

Done